src/sandstorm/run-bundle.c++

```diff
@@ -1183,18 +1189,17 @@ private:
     KJ_SYSCALL(sigprocmask(SIG_SETMASK, &sigset, nullptr));
   }
 
-  void copyEtc() {
+  void linkEtc() {
     auto files = splitLines(readAll("etc.list"));
```

Comment:
Sort of.  It was fine for the initial run because we copy `/etc/resolv.conf`, which derefs to a file with content, but as soon as the `resolv.conf` contents changed, we'd be still using a stale copy.